openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -30,20 +31,27 @@ def count_data():
     # Get IP address of the platform and convert it to latitude, longitude.
     ip_data = requests.get('http://freegeoip.net/json')
     ip_data_json = json.loads(ip_data.text)
-    platform_latitude, platform_longitude = olga_settings.get("PLATFORM_LATITUDE"), olga_settings.get("PLATFORM_LONGITUDE")
+
+    platform_latitude = olga_settings.get("PLATFORM_LATITUDE")
+    platform_longitude = olga_settings.get("PLATFORM_LONGITUDE")
    
     if platform_latitude and platform_longitude:
         latitude, longitude = platform_latitude, platform_longitude
     else:
         latitude, longitude = ip_data_json['latitude'], ip_data_json['longitude']
 
-    # Get students amount within current platform.
-    students_amount = UserProfile.objects.count()
+    # Get count of active students (if logged in during last 30 days).
+    # Built-in edX`s accounts have None in `last_login` field.
+    # 30th day <= Current last login date <= Today
+    activity_border = (datetime.datetime.now() - datetime.timedelta(days=olga_settings.get("ACTIVITY_PERIOD")))
+    active_students_amount = UserProfile.objects.all().exclude(
```

Comment:
Please exclude not active students
e.g. 
```
.exclude(user__is_active=False)
```
Because such students can login but can't access courses
@auraz what do you think?